nservicebus/usage/index.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Measuring system usage
+summary: Measure the usage of an NServiceBus system.
+reviewed: 2024-05-08
+related:
+  - servicepulse/usage
+  - nservicebus/throughput-tool
+---
+
+There are two methods provided for measuring usage of an NServiceBus system:
+
+- [Using ServicePulse](#using-servicepulse)
+- [Standalone Endpoint throughput tool](#standalone-endpoint-throughput-tool)
+
+> [!NOTE]
+> Requirements for both of the usage measuring methods:
+>
+> - NServiceBus Transport: [Azure Service Bus](./../../transports/azure-service-bus), [RabbitMQ](./../../transports/rabbitmq), [Amazon SQS](./../../transports/sqs), [SqlServer](./../../transports/sql) OR
```

Comment:
Probably safer to reference these with [/transports/rabbitmq] rather than relative references, no?